messages/src/lib.rs

```diff
@@ -2353,16 +2320,7 @@ impl Decode for AggregateShareResp {
 
 #[cfg(test)]
 mod tests {
-    use crate::{
-        query_type::{self, FixedSize, TimeInterval},
-        AggregateContinueReq, AggregateContinueResp, AggregateInitializeReq,
-        AggregateInitializeResp, AggregateShareAad, AggregateShareReq, AggregateShareResp,
-        AggregationJobId, BatchId, BatchSelector, CollectReq, CollectResp, Duration, Extension,
-        ExtensionType, FixedSizeQuery, HpkeAeadId, HpkeCiphertext, HpkeConfig, HpkeConfigId,
-        HpkeKdfId, HpkeKemId, HpkePublicKey, InputShareAad, Interval, PartialBatchSelector,
-        PlaintextInputShare, PrepareStep, PrepareStepResult, Query, Report, ReportId,
-        ReportIdChecksum, ReportMetadata, ReportShare, ReportShareError, Role, TaskId, Time,
-    };
+    use super::*;
```

Comment:
Please don't import `*` from a module, for the same reason that one shouldn't do that in e.g. Java or other languages with similar import mechanisms -- doing so makes fixing naming conflicts unnecessarily painful, and can make things more confusing to folks trying to read the code (is type `T` from _this_ star-import or _that_ star-import?).